src/wise_psf.jl

```diff
@@ -0,0 +1,39 @@
+using FITSIO
+
+function wise_psf(band; sidelen=-1)
+
+  # don't believe any sensible normalization is in place, need to fix that ...
```

Comment:
I think there are several possibilities. One way to normalize the PSFs would be to make each PSF have a value of 1 in it's central pixel. Or we could make the sum of the pixels within a certain aperture be 1. Or we could add a keyword argument(s) to allow the user to specify a certain multiplicative scaling of the PSF.